meerk40t/core/elements/elements.py

```diff
@@ -312,6 +316,18 @@ def plugin(kernel, lifecycle=None):
                 "page": "Classification",
                 "section": "_30_GUI-Behaviour",
             },
+            {
+                "attr": "remove_non_used_default_ops",
+                "object": elements,
+                "default": True,
+                "type": bool,
+                "label": _("Remove unused default operations"),
+                "tip": _(
+                    "If a default operation is no longer used it will be removed from the list of active operations"
+                ),
+                "page": "Classification",
+                "section": "_30_GUI-Behaviour",
+            },
```

Comment:
While this isn't bad. I suggestion there was generally to not-show them. You are making a list of operations that you care about. You can just filter out the empty operations you don't care about.

Reply:
I have left the option, but set the default to False and make it visible only in expert mode.
Operation panel will now allow to filter unused ops (default behaviour)

![grafik](https://github.com/meerk40t/meerk40t/assets/2670784/e463f883-691e-4234-a73e-308cd8606bb7)


![grafik](https://github.com/meerk40t/meerk40t/assets/2670784/8b32f3e3-4f2a-4fe5-b96d-ea4d3b7064b2)